stdlib/eq.m31

```diff
@@ -2,20 +2,16 @@
 
```

Comment:
There should be some information about what exceptions a user of this module has to be prepared to deal with, and when they might occur.